tests/Yaapii.Atoms.Tests/Text/ComparableTextTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Yaapii.Atoms.Text.Tests
+{
+    public sealed class ComparableTextTests
+    {
+		[Fact]
+		public void Compares()
+		{
+			Assert.True(
+				new ComparableText(
+					new TextOf("Hallo Welt")
+				).CompareTo(
+					new TextOf("Tschüss Welt")
+				) <= 0 
```

Comment:
You must test for -1, but not for 0.
See: https://msdn.microsoft.com/de-de/library/system.icomparable.compareto(v=vs.110).aspx